rustls/src/crypto/ring/kx.rs

```diff
@@ -102,6 +101,14 @@ impl ActiveKeyExchange for KeyExchange {
     }
 }
 
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn kxgroup_is_debug() {
+        assert_eq!("X25519", format!("{:?}", super::X25519));
+    }
```

Comment:
I'm curios what coverage this is adding. The `Debug` bound is on `SupportedKxGroup` - `KxGroup` doesn't derive `Debug`. Should this be named `supported_kx_group_is_debug`?

Reply:
renamed to `kxgroup_fmt_yields_name`: this test is wanting to observe the behaviour of `KxGroup::fmt`, which is accessed via the Debug trait bound on `SupportedKxGroup`.

Reply:
SGTM!